lib/main.coffee

```diff
@@ -144,6 +144,17 @@ module.exports =
   provideLinter: ->
     helpers = require('atom-linter')
     path = require('path')
+    fs = require('fs')
+
+    applySubstitutions = (path) ->
+      for project in atom.project.getPaths()
+        p = path
```

Comment:
Instead of this mess why not something simple like [this](https://github.com/AtomLinter/linter-pylint/blob/v1.2.0/lib/main.coffee#L78)?